.github/CODEOWNERS

```diff
@@ -1,2 +1 @@
-
-*       @nurdtechie98 @shivam1708 @inishchith @NightWing1998
\ No newline at end of file
+*       @nurdtechie98 @shivam1708 @inishchith @NightWing1998
```

Comment:
I'm not sure if this change is required.

Reply:
I also not sure about that, it is creating merge conflict. That's why I said check CODEOWNERS file. Chirag please confirm.

Reply:
yes I did that